pkg/client/proxy.go

```diff
@@ -95,9 +96,14 @@ func NewProxyClient(ctx context.Context, ctxCancel context.CancelFunc, address s
 }
 
 const (
-	GRPCServiceTimeout = 3 * time.Minute
+	GRPCServiceTimeout = eclient.GRPCServiceTimeout * 2
 )
 
+func (c *ProxyClient) getContextWithGRPCTimeout() context.Context {
```

Comment:
Can we change to a function instead of a method of ProxyClient? and change the name as below.

`createContextForProxyClientGRPCTimeout`

Reply:
Updated.